addons/ttf/ttf.c

```diff
@@ -1005,10 +1005,10 @@ static bool ttf_get_glyph_dimensions(ALLEGRO_FONT const *f,
       }
    }
    cache_glyph(data, face, ft_index, glyph, false);
-   *bbx = glyph->offset_x;
+   *bbx = glyph->offset_x + 1;
    *bbw = glyph->region.w - 2;
```

Comment:
You're right, there's an issue with the offsets bbx and bby. The reason is that gylph->offset_x and glyph->offset_y don't include the extra pixel added to glyph->region.*.
So the return values should really be *bbx = glyph->offset_x and *bby = glyph->offset_y.
(so everything is shifted one pixel forward, accounting for the "error" that you saw in ex_ttf).

Is there a way I can amend this without opening a new pull request?